charts/argus/values.schema.json

```diff
@@ -189,6 +191,13 @@
       ],
       "$comment": "ui:accessKey-ignore tf:optional"
     },
+    "companyDomain": {
+      "$comment": "tf:optional",
+      "$id": "#/properties/companyDomain",
+      "type": "string",
+      "title": "The domain schema",
+      "description": "The Logicmonitor account domain"
```

Comment:
Can we add default and examples for reference.

Reply:
done